client/src/components/AuthFooter/AuthFooter.tsx

```diff
@@ -14,16 +14,9 @@ const AuthFooter = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
   const classes = useStyles();
 
   return (
-    <Box
-      p={1}
-      display="flex"
-      justifyContent="flex-end"
-      alignSelf="flex-end"
-      marginRight={5}
-      className={classes.authHeader}
-    >
+    <Box p={1} display="flex" justifyContent="center" alignSelf="center" marginRight={5} className={classes.authHeader}>
```

Comment:
In v5 MUI, it's preferred to use `sx` now. An example below. With that, we can also just bring in the styles from `authHeader` and put them in `sx` since it's only two CSS properties. Thanks!

```js
<Box sx={{ display: "flex" }}> . . . </Box>
```